src/security.cs

```diff
@@ -337,178 +337,220 @@ interface KeysAccessible {
 		IntPtr WhenPasscodeSetThisDeviceOnly { get; }
 	}
 
-	[Static][Internal]
+	[Static]
 	interface SecAttributeKey {
-		[Mac (10,9)]
+		[iOS (4,0), Mac (10,9), TV (9,0), Watch (2,0)]
 		[Field ("kSecAttrAccessible")]
 		IntPtr Accessible { get; }
 
-		[iOS (8,0), Mac(10,10)]
+		[iOS (8,0), Mac (10,10), TV (9,0), Watch (2,0)]
 		[Field ("kSecAttrAccessControl")]
 		IntPtr AccessControl { get; }
 
-		[iOS (7,0)]
-		[Mac (10,9)]
+		[iOS (7,0), Mac (10,9), TV (9,0), Watch (2,0)]
 		[Field ("kSecAttrSynchronizableAny")]
 		IntPtr SynchronizableAny { get; }
 
-		[iOS (7,0)]
-		[Mac (10,9)]
+		[iOS (7,0), Mac (10,9), TV (9,0), Watch (2,0)]
 		[Field ("kSecAttrSynchronizable")]
 		IntPtr Synchronizable { get; }
 
-		[iOS (9,0)]
-		[NoMac]
+		[iOS (9,0), Mac (10, 12, 4), TV (9,0), Watch(2,0)]
```

Comment:
Don't know how you got the availability attributes but this Mac one (maybe some others) seems incorrect. 

`extern const CFStringRef kSecAttrSyncViewHint    API_AVAILABLE(macos(10.11), ios(9.0));`

Reply:
@mdbech it says `macOS 10.11+` in the link you shared (;

I see `Mac (10, 12, 4)` here, maybe a copy and paste mistake?

Reply:
Yeah, I don't what happen, but I'm fixing it in the next commit.